Bottom: 8c217f9762b4266acdd91615a215d7a71738fd65
Top:    22b3719ed17adb5b9d1fbe60148115d57d6bd5de
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-28 15:46:11 +0100

Refresh of hid-ps-don-t-use-the-core-s-dr

---

diff --git a/src/hid/ps/ps.c b/src/hid/ps/ps.c
index 8042efa..e007d68 100644
--- a/src/hid/ps/ps.c
+++ b/src/hid/ps/ps.c
@@ -469,16 +469,12 @@ ps_expose (void)
   if (set_layer ("bottompaste", SL (PASTE, BOTTOM), paste_empty))
     DrawPaste (SOLDER_LAYER, NULL);
 
-  doing_assy = true;
-
   if (set_layer ("topassembly", SL (ASSY, TOP), 0))
     PrintAssembly (COMPONENT_LAYER, NULL);
 
   if (set_layer ("bottomassembly", SL (ASSY, BOTTOM), 0))
     PrintAssembly (SOLDER_LAYER, NULL);
 
-  doing_assy = false;
-
   if (set_layer ("fab", SL (FAB, 0), 0))
     PrintFab (Output.fgGC);
 
@@ -877,7 +873,7 @@ set_layer (const char *name, int group, int empty)
       strcmp (name, "route") != 0
       )
     {
-      dapi->draw_layer (global.outline_layer, NULL, NULL);
+      DrawLayer (global.outline_layer, NULL);
     }
 
   return 1;
